### DISCUSSION TITLE
Bluetooth sound truncated at the end.

### DISCUSSION THREAD

**Discussion by styrheim** [osmandapp/OsmAnd#13202] [General]
There is a bug somewhere: in my car, my phone, or OsmAnd.

When listening to OsmAnd via Bluetooth in the car, the sound is cut, before the app is finished talking.

E.g. when the app says "You have reached your destination", I hear only "You have reached your DESTINY !".

Funny, but not very useful. Not when it says "…and turn", leaving out the 'left' or 'right'.

OsmAnd; 4.1.6 2021-10-28
Phone: Samsung Galaxy S10+ SM-G975F/DS
Android: 11
Car:  Citroën C-4 2021

I should have tested another phone and another car, I know. But I haven't, not yet.

Has anybody experienced this with other phones, or other car makes?

**Comment by gzuaps** [comment]
Not much help but I've had exactly the same problem since updating my phone about 2 years ago.

Phone:  moto e6 plus
Car: Mitsubishi ASX 2014

The voice prompts will complete if you have music playing over bluetooth at the same time, and will work if not using bluetooth.
I've seen some people suggesting playing a "silent" track at the same time. 
It's possible that changing the Text to Speech engine may fix this issue but I haven't tried that.

**Comment by gzuaps** [comment]
![TTS](https://user-images.githubusercontent.com/9564478/146660124-23ca6708-26a4-4905-9781-1194e3eb982e.png)

In plugins-development-test voice prompts
It looks like you can change voice prompt delay (11.2) to stop voice cutoffs but it doesn't seem to stop cutoffs for me.
I suspect this setting is to prevent cutoffs at the beginning of the output and not for truncation at the end.
